### ISSUE TITLE
Feature Request: Support Vanilla Image Drag Behaviour

### ISSUE THREAD

**Issue by Jordy3D** [Ademking/BetterViewer#49]
**Is your feature request related to a problem? Please describe.**
I'd like to be able to drag the image from the Chrome window directly into Explorer to save it (like you can with almost any draggable/selectable image). 

**Describe the solution you'd like**
The ability to hold a modifier key (Ctrl, Shift, whatever) that would change the drag behaviour to the Chrome default image drag behaviour.

**Describe alternatives you've considered**
A previous Image Viewer extension I used had this feature, though that extension rarely works anymore.

**Additional context**
Basically this behaviour:
![chrome_RYMkulhykF](https://user-images.githubusercontent.com/19144524/212889089-9d795186-5569-413d-9be4-034fa701e03b.gif)
